os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/lib/bt_flags.h

```diff
@@ -0,0 +1,49 @@
+/**

```

Comment:
can you add samsung license top of this

Reply:
After discussion，we think it's better to just stick with Realtek license ,because this is arch and board folder.Thanks.